src/Storage/Storage.Management/Storage.Management.csproj

```diff
@@ -13,6 +13,13 @@
     <AssemblyName>$(AzAssemblyPrefix)$(PsModuleName).Management</AssemblyName>
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="StorageAccount\AddAzureStorageAccountNetworkRule.cs" />
+    <Compile Remove="StorageAccount\GetAzureStorageAccountNetworkRuleSet.cs" />
+    <Compile Remove="StorageAccount\InvokeAzureStorageAccountHierarchicalNamespaceUpgrade.cs" />
+    <Compile Remove="StorageAccount\RemoveAzureStorageAccountNetworkRule.cs" />
+    <Compile Remove="StorageAccount\UpdateAzureStorageAccountNetworkRuleSet.cs" />
```

Comment:
"InvokeAzureStorageAccountFailover.cs" will still fail in build.

Reply:
Thanks for catching that. Excluded it.